dali/pipeline/operators/fused/crop_mirror_normalize.h

```diff
@@ -62,9 +60,15 @@ class CropMirrorNormalize : public Operator<Backend> {
                  "Expected NCHW or NHWC.");
     DALI_ENFORCE(crop_h_ > 0 && crop_w_ > 0);
 
+    GetSingleOrRepeatedArg(spec, &mean_vec_, "mean", C_);
+    GetSingleOrRepeatedArg(spec, &inv_std_vec_, "std", C_);
     // Validate & save mean & std params
-    DALI_ENFORCE((int)mean_vec_.size() == C_);
-    DALI_ENFORCE((int)inv_std_vec_.size() == C_);
+    DALI_ENFORCE((int)mean_vec_.size() == C_,
```

Comment:
Maybe we can put this DALI_ENFORCE directly into GetSingleOrRepeatedArg as we know expected size and argument name there.

Reply:
That's good point, will add that.